sdk/python/kfp/v2/compiler_cli_tests/test_data/pipeline_with_various_io_types.py

```diff
@@ -60,6 +62,7 @@
 - {name: input_e, type: Datasets}
 - {name: input_f, type: Some arbitrary type}
 - {name: input_g, type: {GcsPath: {data_type: TSV}}}
+- {name: input_h, type: HTML}
```

Comment:
Can we also add the Markdown type input here?

Reply:
Not necessarily need to capture all the types we have in this test. 
But `type_utils_test.py` has UTs for the complete list of artifact types.

Reply:
I see, thanks Chen for confirming!